### DISCUSSION TITLE
I started a wiki page

### DISCUSSION THREAD

**Discussion by gyaaniguy** [GravityPDF/querypath#46] [General]
Hi ! Long time querypath user here. Thanks for keeping this live !

I wrote a wiki page in the wiki:  https://github.com/GravityPDF/querypath/wiki/How-to-parse-HTML-in-PHP-using-querypath-library
I copied it from my repo: https://github.com/gyaaniguy/querypath_guide 

This is almost the first time I have tried contributing like this. Please let me know if I am doing something wrong or should do things differently.

Also I would like to contribute to this library. Please feel free to send tasks my way.

**Comment by jakejackson1** [comment]
@gyaaniguy sorry for the delay getting back to you. I've been on leave and only recently returned.

Thanks so much for creating that Wiki article! Any documentation you'd like to contribute is very much appreciated. 🙂

As for how you could further contribute, [I'm about halfway through refactoring the existing examples](https://github.com/GravityPDF/querypath/pull/36/files). If you'd like to help finalize this pull request, it would go along way to help modernize the library. We can then grab snippets from the examples and use them in the plugin README and additional Wiki pages to better document how to use the software.
